src/main/java/ca/corefacility/bioinformatics/irida/util/ToolsListExporter.java

```diff
@@ -158,12 +158,13 @@ public int compare(IridaWorkflow o1, IridaWorkflow o2) {
 					uniqueToolRepositories.add(toolRepository);
 
 					// @formatter:off
-					tools.add(ImmutableMap.<String, String>builder()
+					tools.add(ImmutableMap.<String, Object>builder()
```

Comment:
Modifications are made to this class to print the `tools-list.yml` file in the proper format used by `ephemeris` (tool for automatically interacting/installing software in Galaxy).